packages/restapi/src/lib/space/demoteSpeaker.ts

```diff
@@ -0,0 +1,26 @@
+import type Space from './Space';
+import { SPACE_DISCONNECT_TYPE } from '../payloads/constants';
+
+export interface IDemoteSpeaker {
+    demoteeAddress: string;
+}
+
+export async function demoteSpeaker(
+        this: Space,
+        options: IDemoteSpeaker
+    ) {
+    const { demoteeAddress } = options || {};
+
+    try {
+            this.disconnect({
+                peerAddress: demoteeAddress,
+                details: {
+                    type: SPACE_DISCONNECT_TYPE.DEMOTE_SPEAKER,
+                    data: {},
+                },
+            });
```

Comment:
After disconnect(), we need to update the group data on the BE, demoteeAddress -> isSpeaker -> false
Once this is done, we can fire the meta message here itself.